docs/process/releasing-updates.md

```diff
@@ -110,6 +110,8 @@ Here's what you should do next:
 7. Read this to perform the release: https://github.com/desktop/desktop/blob/development/docs/process/releasing-updates.md
```

Comment:
I think the excerpt we see in our terminal should be improved also -- either to reflect the guidelines or link to them or something in between like
```
3. See the guide for info on how we write notes but, as a quick reminder, make sure to remove internal-facing changes, use the correct tags (list them), and have user-friendly summary lines, and reorder them.

Reply:
great point! done in https://github.com/desktop/desktop/pull/8969/commits/7e4e08cd2c5d7ebb0e173b73eb6aba4a6748bac4